filament/backend/src/Program.cpp

```diff
@@ -45,9 +45,10 @@ Program& Program::setUniformBlock(size_t bindingPoint, utils::CString uniformBlo
     return *this;
 }
 
-Program& Program::setSamplerGroup(size_t bindingPoint,
+Program& Program::setSamplerGroup(size_t bindingPoint, ShaderStageFlags stageFlags,
         const Program::Sampler* samplers, size_t count) noexcept {
-    auto& samplerList = mSamplerGroups[bindingPoint];
+    mSamplerGroups[bindingPoint].stageFlags = stageFlags;
```

Comment:
put `mSamplerGroups[bindingPoint]` in a local, e.g. `auto& group = mSamplerGroups[bindingPoint];`

Reply:
Ok it's done.